### ISSUE TITLE
Consider compression (upgrade of socket.io library)

### ISSUE THREAD

**Issue by rmckeel** [balderdashy/sails.io.js#73]
Please consider upgrading socket.io integration on server and client side to support compression out of the box.

After reviewing a number of ambiguous socket.io GitHub issues and [code](https://github.com/socketio/socket.io/blob/master/lib/socket.js#L158), it appears that socket.io as of ~v1.3.7 is ready to integrate compression.

This could be a gamechanger for WebSocket heavy applications, especially considering how compressable JSON is.

Thoughts?

  Ryan

**Comment by sailsbot** [comment]
Thanks for posting, @rmckeel.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
